porfolio.html

```diff
@@ -16,7 +16,41 @@
             <p class="text">John Doe</p>
         </div>
     </div>
- 
+    <div class="headline">
+        <div class="primary-text">Hey There. <br>
+            I’m Fuad
+        </div>
+        <div class="supporting-text">I’am a Software Developer
+        </div>
+        <div class="description">
+             I can help you build a product , feature or<br>
+             website Look through some of my work <br>
+             and experience! If you like what you see <br>
+             and have a project you need coded, <br>
+             don’t hestiate to contact me.
+        </div>
+        <div class="social-media">
+            <button type="button" disabled class="github">
+                <img src="./images/git.png" alt="a link to github" class="git-icon">
+            </button>
+            <button type="button" disabled class="linkedin">
+                <img src="./images/linkedin.png" alt="a link to linkedin" class="linkedin-icon">
+            </button>
+            <button type="button" disabled class="angle-list">
+                <img src="./images/Enabled.png" alt="a link to angle" class="angle-list-icon">
+            </button>
+            <button type="button" disabled class="twitter">
+                <img src="./images/twitter.png" alt="a link to twitter" class="twitter-icon">
+            </button>
+            <button type="button" disabled class="medium">
+                <img src="./images/m.png" alt="a link to medium" class="medium-icon">
+            </button>
+            </div>
+           <div class="illustration">
+            <img src="./images/illustration-header-mobile.png" alt="" class="illus">
```

Comment:
- Kindly use a background image for this. Decorations are always background images, not `img` tags). If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.
- Ensure that its positioned as it is on the figma template.